packages/expo-dev-launcher/ios/EXDevLauncherBridgeDelegate.mm

```diff
@@ -45,11 +45,7 @@ - (RCTRootView *)createRootViewWithModuleName:(NSString *)moduleName launchOptio
     enableTM = YES;
 #endif
 
-#if REACT_NATIVE_MINOR_VERSION >= 74
```

Comment:
that's my previous fault that `REACT_NATIVE_MINOR_VERSION` is undefined. the `RCTAppSetupPrepareApp(application, enableTM);` is correct code.